src/library_glemu.js

```diff
@@ -3910,5 +3910,7 @@ recordGLProcAddressGet(LibraryGLEmulation);
 
 addToLibrary(LibraryGLEmulation);
 
+#if ASSERTIONS
 assert(!(FULL_ES2 && LEGACY_GL_EMULATION), 'cannot emulate both ES2 and legacy GL');
 assert(!(FULL_ES3 && LEGACY_GL_EMULATION), 'cannot emulate both ES3 and legacy GL');
+#endif
```

Comment:
Are you sure this is needed?  This looks a like a call to the build time `assert` which should always be available, even in release builds.

Reply:
Reverted